docs/modules/rust-guide/pages/multiply-dependency.adoc

```diff
@@ -12,15 +12,15 @@ ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 :sdk-long-name: DFINITY Canister Software Development Kit (SDK)
 :cdk-long-name: DFINITY Canister Development Kit (CDK) for Rust
 
-One common approach to application design is to calculate or store data in one program that you can then use in another program. 
-This ability to share and use functions defined in different application canisters, even if the underlying programs are written in different languages, is an important strategy for building applications to run the {IC}.
+One common approach to Dapp design is to calculate or store data in one smart contract that you can then use in another smart contract. 
```

Comment:
https://en.wikipedia.org/wiki/Decentralized_application uses DApp but lists DApp, dApp, Dapp, and dapp as variants. We should pick one and stick to it :)

Reply:
I prefer just *dapp*, by the way.

Reply:
I agree, we should just pick one.  I am fine with any of them.  I'll watch for the decision and update all the files when the guidelines come out.

Reply:
I changed it to "dapp".  Let's see how that looks.